src/SFA.DAS.EmployerAccounts/Services/ContentApiClient.cs

```diff
@@ -40,7 +42,14 @@ private async Task AddAuthenticationHeader(HttpRequestMessage httpRequestMessage
     {
         if (!string.IsNullOrEmpty(_identifierUri))
         {
-            var accessToken = await _azureClientCredentialHelper.GetAccessTokenAsync(_identifierUri);
+            var azureServiceTokenProvider = new ChainedTokenCredential(
```

Comment:
Would be nicer to inject a service class which implemented getting a token instead of new-ing up here. New is glue etc.